tensorflow/core/ops/array_ops.cc

```diff
@@ -2292,6 +2292,24 @@ Status SliceHelper(InferenceContext* c, ShapeHandle begin_value,
   return Status::OK();
 }
 
+Status SliceHelper(InferenceContext* c, ShapeHandle begin_value,
+                   ShapeHandle sizes_value,
+                   std::vector<DimensionHandle>* dims) {
+  for (int i = 0; i < c->Rank(sizes_value); ++i) {
+    DimensionHandle dim = c->Dim(sizes_value, i);
+    DimensionHandle begin = c->Dim(begin_value, i);
+    if (c->ValueKnown(begin) && c->ValueKnown(dim)) {
+      // If both `begin` and `size` are known, processing:
```

Comment:
c->Subtract should handle the case of when either dim is unknown, so the if/else can be replaced with just the  contents of the if block.

Reply:
Addressed. Thanks.